examples/interdomain/usecases/FloatingKernel2Wireguard2Kernel/README.md

```diff
@@ -58,12 +58,17 @@ metadata:
   name: nse-kernel
 spec:
   template:
+    metadata:
+      annotations:
+        registration-name: icmp-server@my.cluster3
```

Comment:
Why are we adding an annotation for registration-name here?

Reply:
I guess there is a problem with kustomize merge. OK, we'll look at a better solution for it.

Reply:
What is the problem with kustomize merge?

Reply:
@edwarnicke See at https://github.com/networkservicemesh/deployments-k8s/pull/4057